drivers/flash/Kconfig.at45d

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2020 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig SPI_FLASH_AT45D
+	bool "AT45D family DataFlash driver"
+	select FLASH_HAS_DRIVER_ENABLED
+	depends on SPI
+	help
+	  This driver can handle several instances of AT45D family chips that
+	  are enabled by specifying devicetree nodes with the "compatible"
+	  property set to "adesto,at45d" and other required properties like
+	  JEDEC ID, chip capacity, block and page size etc. configured
+	  accordingly.
+
+	  The driver is only capable of using "power of 2" binary page sizes
+	  in those chips and at initialization configures them to work in
+	  that mode (unless it is already done).
+
+if SPI_FLASH_AT45D
+
+config SPI_FLASH_AT45D_USE_READ_MODIFY_WRITE
```

Comment:
Why this is no default y? This is my expectation to any flash drive and I believe most of others expectation as well..

Reply:
@anangl?

Reply:
Oh, right, I modified the sample accordingly but forgot to add this actual `default y` here. Sorry for that. Fixed now.